--- venv/lib/python3.5/site-packages/matplotlib/backends/backend_pdf_ORIG.py
+++ venv/lib/python3.5/site-packages/matplotlib/backends/backend_pdf.py
@@ -1711,11 +1711,22 @@
     def draw_markers(self, gc, marker_path, marker_trans, path, trans,
                      rgbFace=None):
         # Same logic as in draw_path_collection
         len_marker_path = len(marker_path)
         uses = len(path)
         if len_marker_path * uses < len_marker_path + uses + 5:
             RendererBase.draw_markers(self, gc, marker_path, marker_trans,
                                       path, trans, rgbFace)
             return
+
+        try:
+            import point_union
+        except:
+            pass
+        else:
+            p = point_union.compress_markers(
+                marker_path, marker_trans, path, trans)
+            if p:
+                self.draw_path(gc, p, trans, rgbFace)
+                return

         self.check_gc(gc, rgbFace)
